sanic/app.py

```diff
@@ -459,7 +459,7 @@ def enable_websocket(self, enable=True):
     def blueprint(
         self,
         blueprint: Union[
-            Blueprint, List[Blueprint], Tuple[Blueprint], BlueprintGroup
+            Blueprint, List[Blueprint], Tuple[Blueprint, ...], BlueprintGroup
```

Comment:
We probably could collapse to sequence. IIRC, we just loop thru them, so a set and a frozenset would also be valid.

Reply:
Sequence is not suitable for set and frozenset, so I replaced Union[list, tuple] with Iterable. This will allow the use of any object that returns a Blueprint when iterated, including set and frozenset.